Visi.Repository/Visi.Repository.csproj

```diff
@@ -5,11 +5,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.3" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="2.2.3" />
-    <PackageReference Include="Vonk.Core" Version="3.1.1" />
-    <PackageReference Include="Vonk.Facade.Relational" Version="3.1.1" />
-    <PackageReference Include="Vonk.Fhir.R3" Version="3.1.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.0">
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
very correct to add this privately. Did VS do this by itself?

Reply:
Yes this was all VS and the NuGet package manager